README.md

```diff
@@ -1 +1,153 @@
+A *disagreement* between reviewer1 and reviewer2 is an applicant for whom the two reviewers gave diverging grades, i.e.`1` and `-1`. After grading all motivations, you may want to go through all disagreements with other reviewers to check if you simply made a typo or if you really disagree. In case of a real disagreement, a discussion among reviewers will be needed. To go through all disagreements with reviewer `reviewer2` enter grading mode like this:
+
+```
+grader> grade -d reviewer2
+```
+
+This will allow you to change or keep your grade for the corresponding motivation letter.
+
+... repeat for all reviewers.
+
+## Use case: setting overrides
+While grading you may notice that an applicant has misjudge for example their Python proficiency, by self-rating as `novice/advanced-beginner`, when instead you judge them to be rather `competent/proficient`. In this case you should set an override for this field in their application. You type `o python` while in grading mode.
+
+Note that in grading mode you have tab-completion on applicant fields. Once the field is auto-completed, by hitting `<tab>` again you'll get a list of possible values to select from. This will set a permanent override for this applicant in the `applications.ini`
+
+## Use case: ranking applications
+After all reviewers have completed the motivation letter grading, it is possible to *rank* the applications. Each applicant gets a *score* calculated by evaluating a *formula*. 
+
+You can get the formula used for ranking with: `grader> formula`.
+
+You can visualize the ranking of the applicants with: `grader> rank`.
+
+Highlanders will be visualized on top of other applicants.
+
+## Use case: viewing individual applications
+From the grader prompt you visualize applications by using the `dump` command. This allows you for example to visualize a subset of applications matching certain criteria. More details with `dump -h`.
+
+For example, if you want to visualize all applicants with Italian nationality, you type:
+```
+grader> dump -a nationality Italy 
+```
+
+You can get a list of possible attributes with `dump -a list`
+
+## Use case: show stats
+You can get some stats about applicants with the command `stat`. More details with `stat -h`.
+
+## Use case: dynamically change the formula
+You can experiment with effect of different weights in the formula on the applicant ranking by changing the formula as in this example:
+
+```
+grader> formula programming*0.2 + open_source*0.2
+```
+
+This will print the contributions of each factor to the final score of an applicant:
+
+```
+formula = programming*0.2 + open_source*0.2
+score ∈ [ 0.000, 0.400]
+applied ∈ [0, 1, 2]
+contributions:
+programming*0.2 : 50.0%
+open_source*0.2 : 50.0%
+```
+
+By ranking the applicants using the `rank` command you will visualize the new ranking based on the new formula.
+
+To make this change permanent, you need to set the formula with:
+
+```
+grader> formula -s programming*0.2 + open_source*0.2
```

Comment:
This goes against the design of the rest of grader, all state changes are "permanent" in the sense that they are saved to `applications.ini` on exit. The user is expected to have the files under git.